### ISSUE TITLE
TIME error messages with wrong number of args

### ISSUE THREAD

**Issue by danielcompton** [rethinkdb/rethinkdb#4755]
If you pass 1 argument to time, you get the error message `Expected between 4 and 7 arguments but found 1`. If you pass 5 arguments then you get the error message `Got 5 arguments to TIME (expected 4 or 7).`

I propose two improvements to the error message:
1. Make the first error `Expected 4 or 7 arguments but found 1`.
2. Include the query term and arguments in the error message, e.g. `TIME expected 4 or 7 arguments but found 1: <print arg>`.

**Comment by danielmewes** [comment]
That's definitely inconsistent, and I agree that the "between" is misleading.

I'm not sure if we list the arguments anywhere else, do we?

**Comment by danielcompton** [comment]
Yeah, including the arguments might be a step too far for this issue.
